src/main/scala/com/databricks/labs/overwatch/pipeline/GoldTransforms.scala

```diff
@@ -279,7 +279,7 @@ trait GoldTransforms extends SparkSessionWrapper {
 
     val derivedSKU = PipelineFunctions.deriveSKU(isAutomated('cluster_name), 'spark_version, 'cluster_type)
     val nodeTypeLookup = clusterSpec.asDF
-      .select('organization_id, 'cluster_id, 'cluster_name, 'custom_tags, 'timestamp, 'driver_node_type_id, 'node_type_id, 'spark_version, 'cluster_type, 'runtime_engine)
+      .select('organization_id, 'cluster_id, 'cluster_name, 'custom_tags, 'timestamp, 'driver_node_type_id, 'node_type_id, 'spark_version, 'cluster_type, 'runtime_engine,'num_workers)
```

Comment:
This needs to be pulled from clusterSnapshot not clusterSpec. ClusterSpec will also not have the num_workers populated since it's source is also the audit logs. The ticket clearly suggests getting this value from cluster_snapshot not clusterspec; thus this logic should be applied to the `nodeTypeLookup2`, not `nodeTypeLookup` as done here.